provider/schema/table.go

```diff
@@ -80,6 +82,40 @@ func (tco TableCreationOptions) signature() string {
 	return strings.Join(tco.PrimaryKeys, ";")
 }
 
+// IsIgnoreError returns true if the error is ignored via the current table IgnoreError function or in any other parent table (in that ordered)
+// it stops checking the moment one of them exists and not until it returns true or fals
+func (t Table) IsIgnoreError(e error) bool {
+	// first priority is to check the tables IgnoreError function
+	if t.IgnoreError != nil {
+		return t.IgnoreError(e)
+	}
+	// secondy priority is to check the parent tables IgnoreError recursively
+	if t.Parent != nil {
+		return t.Parent.IsIgnoreError(e)
+	}
+
+	return false
+}
+
+// IsIgnoreErrorColumn returns true if the error is ignored via the column resolver, current table IgnoreError function or in any other parent table (in that ordered)
+// it stops checking the moment one of them exists and not until it returns true or fals
+func (t Table) IsIgnoreErrorColumn(c Column, e error) bool {
```

Comment:
What is the use case for running the table ignore error function for specific columns?
Are we expecting similar errors to happen?

When I added an ignore error for columns it was super specific to the data type of the column and the provider API:
https://github.com/cloudquery/cq-provider-gcp/blob/49238085984bc25468ced7484761a3ceb1b7a99e/client/error.go#L59

I'd expect that table level errors are more about failures to fetch the data, and column level errors are more about failures to parse/normalize the data

Reply:
Where do we use ColumnIgnoreError apart from this case? trying to find it with regex...

Reply:
I think I might have been the first consumer of that feature as I had to do another fix in our testing logic to get it to work https://github.com/cloudquery/cq-provider-sdk/pull/283

Reply:
I see. so seems to me it will be best to remove this feature as well as it is unused, so less documentation, less bugs. and given this PR was the first consumer I suggest to implement it by wrapping up `IPNetResolver` with `GCPIPNetResolver` that will also ignore this error. WDYT ?

Reply:
So why not just only recursively propagate the table level error handler?

Reply:
Ah well, that's how I started and the realized we have in the SDK. I'll open a new issue for that.

Reply:
Or even better add to the `IPNetResolver` a check that if it's empty just don't do anything. seems to me like a good default behaviour for all cases or just set it to null instead. both seems good to me.

Reply:
Opened https://github.com/cloudquery/cq-provider-sdk/issues/327 and https://github.com/cloudquery/cq-provider-sdk/issues/326.

Didn't want to change the default `IPNetResolver` back then as I only had a very specific use case for the GCP provider. Didn't see empty strings coming from other APIs.

Reply:
I actually already pushed it as it's less code in this PR, so I just need to add a test now